### ISSUE TITLE
Option to hide boosts/replies

### ISSUE THREAD

**Issue by jespercockx** [elk-zone/elk#2435]
I just found out about the existence of this project and it's great to have a really well-designed and uniform interface to access my different Fediverse accounts from the browser. One feature that I miss from the standard Mastodon UI is the ability to hide replies and boosts from my timeline, and just view the new posts made by the people I follow. I find this feature very helpful when I just want to check quickly what is going on without getting sucked into the latest 50+ reply discussion or have to scroll through the 100 boosts that someone felt like they needed to make.

**Comment by SteveDinn** [comment]
I don't care to hide boots altogether, but it would be nice if there was at least some time between when you saw the same post.

For example, I often see this sort of thing. A post followed (often directly after) by a boost of the same.

![image](https://github.com/elk-zone/elk/assets/3911507/973a7061-e1f8-42ab-9e4f-b41208053014)

**Comment by rstub** [comment]
@SteveDinn Do you have "Preferences -> Other -> Group boosts in timelines" enabled within your *standard* Mastodon settings?

**Comment by SteveDinn** [comment]
@rstub Yes, I have that option checked. This problem does not exist in the native Mastodon web interface.

**Comment by Karmalakas** [comment]
I was searching for an alternative for mastodon.social and found Elk. Started the search, because on official page I still can see most of replies, even though I checked to hide them. For example Tusky app on Android doesn't show them. Elk still does show and that's a no go for me :(

**Comment by Karmalakas** [comment]
Wonder if there's any chance this could be implemented... 🙄

**Comment by kpocius** [comment]
Seems like this request duplicates https://github.com/elk-zone/elk/issues/673 which has a lot more attention.
